import-automation/executor/app/executor/import_executor.py

```diff
@@ -351,6 +351,78 @@ def _get_latest_version(self, import_dir: str) -> str:
         latest_version = blob.download_as_text()
         return f'gs://{bucket.name}/{import_dir}/{latest_version}'
 
+    def _get_import_input_files(self, import_input, absolute_import_dir):
+        input_files = []
+        template_mcf = []
+        if pattern := import_input.get('template_mcf'):
+            template_mcf = glob.glob(os.path.join(absolute_import_dir, pattern))
+            import_prefix = os.path.splitext(os.path.basename(
+                template_mcf[0]))[0]
+            input_files.extend(template_mcf)
+
+        cleaned_csv = []
+        if pattern := import_input.get('cleaned_csv'):
+            cleaned_csv = glob.glob(os.path.join(absolute_import_dir, pattern))
+            input_files.extend(cleaned_csv)
+
+        node_mcf = []
+        if pattern := import_input.get('node_mcf'):
+            node_mcf = glob.glob(os.path.join(absolute_import_dir, pattern))
+            input_files.extend(node_mcf)
+            if not import_prefix:
+                import_prefix = os.path.splitext(os.path.basename(
+                    node_mcf[0]))[0]
+        return input_files, import_prefix
+
+    def _invoke_import_tool(self, absolute_import_dir: str,
+                            relative_import_dir: str, version: str,
+                            import_spec: dict):
```

Comment:
can this return a list of validation dirs or import_prefixes for the next stage?